pages/common/hut.md

```diff
@@ -0,0 +1,29 @@
+# hut
+
+> A CLI tool for sourcehut.
+> Use `hg` in place of `git` below if using mercurial.
```

Comment:
```suggestion
> Note: Use `hg` in place of `git` below if using Mercurial.
```
We prefer highlighting the capitalized version of Brand/Project names wherever applicable.